assignment/tool/src/java/org/sakaiproject/assignment/entityproviders/AssignmentEntityProvider.java

```diff
@@ -675,6 +675,34 @@ private Map<String, Object> submissionToMap(Set<String> activeSubmitters, Assign
         if (as.getGraded()) submission.put("graded", as.getGraded());
 
         if (hydrate) {
+
+            // Add the SubmissionReview Launch information if this tool has requested it
+            // https://www.imsglobal.org/spec/lti-ags/v2p0#submission-review-message
+            Set<AssignmentSubmissionSubmitter> submitters = as.getSubmitters();
+            if ( submitters.size() == 1 ) {
```

Comment:
should use the assigment isGroup to determine this

Reply:
I changed the approach to use submittee and then falling back to the single submittee.  Please re-review.

Reply:
Hey @ern - I want to keep the scope of this PR to fix the issue that the JIRA identified.  It has a nice test plan and is easy to verify.  Since the group support for LTI Group assignments was never developed nor tested before Sakai-25.  There are a lot of JIRAs that already exist that document needed improvements to LTI Group assignments.  And while this fix might include a partial fix to those other JIRAs - I don't want to expend this fix to attempt to expand this so as to roll up all the other JIRAs.  I want to get the fix for this JIRA as good as it can be, then merge it and test / verify it then move on to other JIRAs.

Reply:
understood, updated your PR with some simpler logic around the submitter

Reply:
@ern I pulled down you r change and it worked the same as my code, suggest approve and merge.